setup.py

```diff
@@ -33,8 +33,10 @@
     ],
     author='Isaac Murchie',
     author_email='isaac@saucelabs.com',
+    maintainer='Kazuaki Matsuo, Mykola Mokhnach, Mori Atsushi',
     url='http://appium.io/',
-    packages=find_packages(include=['appium*']),
+    package_data={'appium': ['webdriver/py.typed']},
```

Comment:
👍 

Thanks, I missed this at https://github.com/appium/python-client/pull/508.